foubus.py

```diff
@@ -358,19 +359,34 @@ def term_write(s):
         html.write(f'  <div class="label">{trip_label}</div>\n')
         term_write(f'{trip_label:15.15} ')
         print(rt)
+        # The following code includes creative contributions from Claude, a generative AI system.
+        # https://declare-ai.org/1.0.0/total.html
         if rt:
             (r,) = rt  # assert len 1
-            delta = int(r.leave_in.total_seconds()) // 60
+            total_seconds = int(r.leave_in.total_seconds())
+            if total_seconds < 60:
+                delta_display = f"{total_seconds} sec"
```

Comment:
I'm not sure that "seconds" is a meaningful granularity here. Leaving in "10 seconds" and leaving in "20 seconds" is approximately the same thing.

Now that you have the board dynamically updating every 1 second, once this reaches N seconds, it will become a constant distraction on the screen, changing every second for 60 seconds. (overstimulating, anxiogenic, etc)

I would really prefer to have the finest granularity be "minutes". Are you strongly attached to "seconds"?